lib/routes/seagullApi.js

```diff
@@ -302,10 +304,6 @@ module.exports = function (crudHandler, userApiClient, gatekeeperClient, metrics
               }
             });
             userProfiles = _.flatten(results);
-
-            if (userProfiles.length !== userIds.length) {
-              throw new Error('Received different number of results than we queried for');
-            }
           } catch (error) {
             res.send(500);
```

Comment:
Is it possible to get here now?  If so, we should log an error message.  If not, we should delete this code path.

Reply:
It's possible if `const users = await getUsers(usersChunk);` throws an exception - I'll get that fixed.

Reply:
This was resolved after rebasing master